src/platform_impl/linux/taskbar/unity.rs

```diff
@@ -0,0 +1,113 @@
+use dbus::{
```

Comment:
I don't quite like this implementation to be honest since it depends on dbus and I think we can get the same effect using dlopen/dlsym similar to electron implementation here https://github.com/electron/electron/blob/905aad9cb6007c2e0ff4897ec64f0c9967f63ee2/shell/browser/linux/unity_service.cc#L134

Reply:
hmm...
Any specific reason for not preferring dbus actually?

Reply:
Asking because to be honest, I am a full newbie at linux apis

Reply:
Thanks, I shall also try to port it to zbus or dlopen or dlsym using my wsl and ubuntu vm

Reply:
@amrbashir Shall I change it to zbus?

Reply:
yeah sure, go ahead

Reply:
Actually it seems like dbus is smaller than zbus (about 43kb bigger)
Shall we keep it dbus? or shall I shift to zbus?

Reply:
@amrbashir Shall it use dbus?

Reply:
@amrbashir

Reply:
I'd like to use zbus anyways, because tauri uses `rfd` crate which uses `zbus` too and then there is `tauri-plugin-single-instance` which uses `zbus` as well so If we were to go with the dbus approach, we should be using `zbus`

Reply:
Implemented something with zbus with wsl green signal. Idk if it'll work